lib/choice.fz

```diff
@@ -83,8 +83,8 @@
 #
 # When assigning values to temp, we need to wrap the values accordingly:
 #
-#   set temp := centigrade 37
-#   set temp := fahrenheit 99
+#   temp := centigrade 37
+#   temp := fahrenheit 99
```

Comment:
The assignment to `temp` here declares new variables with the same name, so they are of type `centrigrade` or `fahrenheit`, not a choice type. 
Can you change this example to declare `temp` as an argument, e.g., 

```
hasFever(tmp centigrade | fahrenheit) bool is ...
```
and then do 
```
  hasFever (centrigrad 37)
  hasFever (fahrenheit 99)  
```